### PR TITLE
Update for mysql8 new defaults behaviour

### LINKED ISSUE #90: Correct for mysql8 default behaviour changes
ref: https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #90

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/MySQL.php [hunk 1]
```diff
@@ -234,13 +234,9 @@ public function getTables(?array $tables = null): array
             }
 
             if ($column['EXTRA']) {
-                $curColumn["extra"] = $column["EXTRA"];
                 if ($column['EXTRA'] === 'auto_increment' && isset($tableDefs[$curTable]['autoIncrement'])) {
                     $curColumn['autoIncrement'] = $tableDefs[$curTable]['autoIncrement'];
                 }
-                if ($column['EXTRA'] === 'on update CURRENT_TIMESTAMP' && $column['COLUMN_DEFAULT'] === 'CURRENT_TIMESTAMP') {
```
Comment (odinuv) [inline src/Keboola/DbExtractor/Extractor/MySQL.php]: isn't this used for incremental fetching?

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: nope, never used

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: it used to be used, but then we decided to let them use any timestamp or datetime column they want and use it at their own risk

**Comment by pivnicek** [comment]
Oh, I made a new issue for that, didn't see that there already was one.

**Pull request merged**